package.json

```diff
@@ -57,7 +57,7 @@
   },
   "homepage": "https://github.com/exadel-inc/ui-playground#readme",
   "dependencies": {
-    "@exadel/esl": "^2.8.1",
+    "@exadel/esl": "^3.0.0-beta.14",
```

Comment:
Please update to the latest one. There was a couple of more releases.

Reply:
thanks, fixed :)